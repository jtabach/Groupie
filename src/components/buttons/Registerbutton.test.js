import React from 'react';
import renderer from 'react-test-renderer';
import { StaticRouter } from 'react-router';
import RegisterButton from './RegisterButton';

test('RegisterButton Link matches snapshot', () => {
  const component = renderer.create(
    <StaticRouter location='/' context={{}}>
      <RegisterButton />
    </StaticRouter>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
