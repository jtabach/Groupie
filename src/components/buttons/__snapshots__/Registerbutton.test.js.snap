// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisterButton Link matches snapshot 1`] = `
<a
  href="/register"
  onClick={[Function]}
>
  Register
</a>
`;
